docs/book/src/developer/architecture/controllers/support-multiple-instances.md

```diff
@@ -0,0 +1,41 @@
+# Support running multiple instances of the same provider
+
+Up until v1alpha3, the need of supporting multiple credentials was addressed by running multiple
```

Comment:
Might make sense to also add a note to https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md

Reply:
Done, PTAL